src/main/java/com/courtcanva/ccfranchise/config/SecurityConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.courtcanva.ccfranchise.config;
```

Comment:
this one is duplicated with another PR, right?

Reply:
already merge